src/app/pages/system/general-settings/localization-form2/localization-form2.component.html

```diff
@@ -0,0 +1,11 @@
+<mat-card class="form-card">
+  <div class="mat-content">
+    <form #f="ngForm" (ngSubmit)="onSubmit(f)" class="form-wrap">
+      <ix-form-field-set>
+        <ix-input name="myInput" id="myInput" [placeholder]="'My Name'" [required]="true" ngModel></ix-input>
+        <ix-select [options]="selectOptions" [placeholder]="'My Sibling'" name="mySelect" id="mySelect" ngModel></ix-select>
```

Comment:
This looks pretty good, I only have two complaints:
1. We should use reactive forms.
2. The resulting components may still be too fat. For example, ix-input has `maskValue` functionality, error handling, show/hide password toggle, etc. This is definitely better than what we currently have, but will also create problems in the future, where if you want different functionality you will have to open base component and add another @Input().
Take a look at https://material.angular.io/components/input/examples and see if we can use a similar approach of having more composable components. We don't have to do it now, if it's too complicated though.